<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsteroidsGame</class>
 <widget class="QMainWindow" name="AsteroidsGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MainWidget" name="gameBoard" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>802</width>
      <height>602</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionExit"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/assets/icons/start.png</normaloff>:/assets/assets/icons/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/assets/icons/exit.png</normaloff>:/assets/assets/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/assets/icons/about.png</normaloff>:/assets/assets/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>mainwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
